lib/Elasticsearch/Tests/QueryConditionGeneratorTest.php

```diff
@@ -42,43 +60,46 @@ public function it_generates_a_structure_of_root_level_fields()
     {
         $condition = $this->createCondition()
             ->field('id')
-                ->addSimpleValue(2)
```

Comment:
These need to be reverted. Builder don't need to be on the same level.

Reply:
Agree, but that makes using PHP CS Fixer impossible?

Reply:
I ran the php-cs-fixer locally (using the makefile, and updated Docker image) but there were no reported problems 👩‍🔬 are you using a different version?

Reply:
I'm just running `make`, don't know anything about anything. :laughing:

Reply:
Also, reverted the style changes.

Reply:
Can you apply the correct style on the new code? 😅 Also if there is a specific fixer that's causing this please let me know so I can have a look 😛

Reply:
I thought I did, this was also done by CS Fixer. :see_no_evil: 

I got the original coding style (and this one) by just running `make`, it did the whole thing and came up with this. Haven't run anything other than that, so whatever the current setup does, that's what it did.

Reply:
@sstok any idea what to do here? If I apply CS style with CS fixer, I'll get the stuff we reverted again. :(

Reply:
Disable the CS fixer for now 😛 I will look into it. Btw, isn't this caused by PHPStorm? As far as I know php-cs-fixer should leave this things alone.

Reply:
Nope, I'm not running style changes from PhpStorm anymore, it's done by CS Fixer 100%.

If I disable CS Fixer, how do I apply correct CS here? :rofl:

Reply:
By hand 😄 if you still know how to do this :trollface: Endless comments on what needs to be done different 🤣

Reply:
Of course, but I don't know what the actual CS for this project is, this is the trap of automation, once the robots let you down, you're left sitting in sand wondering where did it all go wrong. :laughing: